build.gradle

```diff
@@ -347,10 +347,14 @@ subprojects {
         dependsOn project.getTasksByName('publishMavenJavaPublicationToMavenLocal', true)
         classpath = configurations.linkageChecker
         main = 'com.google.cloud.tools.opensource.classpath.LinkageCheckerMain'
-        args '-a', "com.google.api:${project.name}:${project.version}",
-            '--exclusion-file', 'linkage-checker-exclusion.xml'
-        // The exclusion file above can be regenerated by '-o' option. See its Wiki for details:
-        // https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/LinkageCheckerMain#exclusion-files
+
+        def arguments = ['-a', "com.google.api:${project.name}:${project.version}"]
+        if (project.name == 'gax-grpc') {
+          // The exclusion file can be regenerated by '-o' option. See its Wiki for details:
+          // https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/LinkageCheckerMain#exclusion-files
+          arguments += ['--exclusion-file', 'linkage-checker-exclusion.xml']
```

Comment:
This makes me wonder if we really need to set the name and have a command line argument at all. Does it make sense to have a standard filename and location where the linkage checker looks?